content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -168,37 +168,39 @@ Once a feature gate goes GA its value becomes locked to `true` by default.
 Feature gate descriptions:
 
 `ControlPlaneKubeletLocalMode`
-: With this feature gate enabled, when joining a new control plane node, kubeadm will configure the kubelet
-to connect to the local kube-apiserver. This ensures that there will not be a violation of the version skew
-policy during rolling upgrades.
+: With this feature gate enabled, when joining a new control-plane node, kubeadm will configure the kubelet
+  to connect to the local kube-apiserver. This ensures that there will not be a violation of the version skew
+  policy during rolling upgrades.
```

Comment:
i think we can avoid this indentation on the FGs.
does it improve the rendering somehow?

Reply:
Both have the same rendering effect. Only provide some convenience with a clean editing look and later change, similar to those bullet text.

Reply:
i prefer without indent as it's easier to edit, but deferring to docs approvers on what is better.

Reply:
other than that, lgtm.
thanks for the pr.